<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="PI-27" failures="1" tests="1" name="Error Validation" time="3.06" errors="0" timestamp="2024-11-19T14:45:42 IST">
  <testcase classname="PI.admin.ErrorValidationTest" name="SubmitOrder" time="0.766">
    <failure type="java.lang.AssertionError" message="expected [Incorrect credentials or wrong password] but found [Incorrect credentials or wrong pass1@word]">
      <![CDATA[java.lang.AssertionError: expected [Incorrect credentials or wrong password] but found [Incorrect credentials or wrong pass1@word]
at PI.admin.ErrorValidationTest.SubmitOrder(ErrorValidationTest.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- SubmitOrder -->
  <testcase classname="PI.admin.ErrorValidationTest" name="candidateErrorValidation" time="0.0">
    <ignored/>
  </testcase> <!-- candidateErrorValidation -->
</testsuite> <!-- Error Validation -->
